console/console-init/ui/src/modules/connection-detail/components/ConnectionLinksToggleGroup/ConnectionLinksToggleGroup.tsx

```diff
@@ -100,14 +100,16 @@ const ConnectionLinksToggleGroup: React.FunctionComponent<IConnectionLinksToggle
     <>
       <ToolbarItem spacer={{ md: "spacerNone" }} data-codemods="true">
         <ToolbarFilter
+          id="connection-links-togglegrp-name-filter"
           chips={selectedNames.map(filter => filter.value)}
           deleteChip={onDelete}
           categoryName="Name"
         >
           {filterSelected && filterSelected.toLowerCase() === "name" && (
             <InputGroup>
               <TypeAheadSelect
-                id="cl-filter-select-name"
+                id="connection-links-togglegrp-name-typeahead"
```

Comment:
Again typeahead is too PF specific, instead names like dropdown or autopopulate which are more generic, should be used.